src/platform_impl/linux/x11/event_processor.rs

```diff
@@ -37,6 +37,7 @@ pub(super) struct EventProcessor<T: 'static> {
     pub(super) first_touch: Option<u64>,
     // Currently focused window belonging to this process
     pub(super) active_window: Option<ffi::Window>,
+    pub(super) key_event_times: [ffi::Time; 256],
```

Comment:
Perhaps `256 - KEYCODE_OFFSET` given that we know the last 8 entries will never be used?